ansible_collections/arista/cvp/plugins/modules/cv_container.py

```diff
@@ -141,10 +146,10 @@ def get_root_container(containers_fact, debug=True):
         Name of the root container, if not found, return Tenant as default value
     """
     for container in containers_fact:
-        logging.debug('  -> CloudVision container %s', str(container))
+        MODULE_LOGGER.debug('  -> CloudVision container %s', str(container))
         if container['Key'] == 'root':
             # if debug:
-            logging.debug('  -> CloudVision ROOT container has name %s', container['Name'])
+            MODULE_LOGGER.info('  -> CloudVision ROOT container has name %s', container['Name'])
```

Comment:
should this be MODULE_LOGGER.debug?

Reply:
I thought it was something we may want to know even if we are not running in debug mode with a lot of outputs.

But we can definitely change it to debug if you think it is not relevant outside of a large debug output.